easybuild/easyconfigs/g/GPAW/GPAW-19.8.1-foss-2018b-ASE-3.18.0-Python-3.6.6.eb

```diff
@@ -0,0 +1,46 @@
+easyblock = "PythonPackage"
+
+name = 'GPAW'
+version = '19.8.1'
+pythonsuffix = '-Python-%(pyver)s'
+aseversion = '3.18.0'
```

Comment:
@schiotz You'll need to rename these local variables to `local_pythonsuffix` and `local_aseversion`, see https://easybuild.readthedocs.io/en/develop/Easyconfig-files-local-variables.html for more information

Reply:
took the liberty to fix this myself in 985f5bae78, so we can get this merged...